api/app.test.js

```diff
@@ -25,6 +25,34 @@ describe("POST /seedDatabase", () => {
   });
 });
 
+describe("POST /entry", () => {
+  it("adds an item to the database", async () => {
+    await supertest(app)
+      .post("/entry")
+      .send({
+        eventId: "123",
+        creatorId: "456",
+        title: "Happy day",
+        description: "and a happy night too",
```

Comment:
Let's add start time and end time to the test data here, as a means of documenting what the properly type / format is.